### PR TITLE
Feature/minor code fixes

### PR DESCRIPTION
**Pull request by anz-gordonj7**
* Add my local_ci.sh script which everyone should be using (or at least something similar) so we dont get golint-ci warnings in pull requests!

* Setup a generate.go file for the grammar output

@anz-bank/sysl-developers

### CODE REVIEW

File: sysl2/sysl/exporter/type_exporter.go [hunk 1]
```diff
@@ -51,63 +51,58 @@ func (t *TypeExporter) populateTypes(syslTypes map[string]*proto.Type, swaggerTy
...
-				attrValueMap, err = t.findSwaggerType(attV)
+				attrValueMap, err := t.findSwaggerType(attV)
 				if err != nil {
 					t.log.Warnf("Type matching failed %s", err)
 					return err
 				}
+				elementSchema.Format = attrValueMap.Format
+				elementSchema.Type = []string{attrValueMap.Type}
 			}
-			elementSchema.Format = attrValueMap["Format"]
```
Comment (anz-gordonj7) [inline sysl2/sysl/exporter/type_exporter.go]: @springwiz this is why maps are bad...

Comment (springwiz) [reply sysl2/sysl/exporter/type_exporter.go]: @anz-gordonj7 You mean missing type info in yml is due to maps being used?

Comment (anz-gordonj7) [reply sysl2/sysl/exporter/type_exporter.go]: no, the key is incorrect. "**f**ormat" was used everywhere else, but this one used "**F**ormat"

Comment (springwiz) [reply sysl2/sysl/exporter/type_exporter.go]: That is a "typo", I could have used constants. It still doesn't prove maps are bad.

Comment (springwiz) [reply sysl2/sysl/exporter/type_exporter.go]: Thanks for fixing :-)

**Pull request merged**